rclpy/rclpy/handle.py

```diff
@@ -1,136 +0,0 @@
-# Copyright 2019-2020 Open Source Robotics Foundation, Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-from threading import Lock
-
-from rclpy.impl.implementation_singleton import rclpy_implementation as _rclpy
-
-
-InvalidHandle = _rclpy.InvalidHandle
```

Comment:
I think this needs to be kept around because it still gets raised [when trying to use a destroyed `Destroyable`](https://github.com/ros2/rclpy/blob/b3cd14d65d4aeb528b6bcd2db489c8000acfaf82/rclpy/src/rclpy/destroyable.cpp#L37). How about we move it to this file? https://github.com/ros2/rclpy/blob/master/rclpy/rclpy/exceptions.py